frontend/src/scenes/onboarding/Personalization.tsx

```diff
@@ -20,46 +19,37 @@ function _Personalization(): JSX.Element {
     return (
         <Row className="personalization-screen">
             <Col xs={24}>{step === null && <StepOne />}</Col>
-            <Col xs={24}>{step === 2 && <StepTwo />}</Col>
         </Row>
     )
 }
 
 function StepOne(): JSX.Element {
-    const { push } = useActions(router)
-    return (
-        <div>
-            <h2 className="subtitle">Nothing to see here yet! Tap Continue</h2>
-            <Button type="primary" onClick={() => push('/personalization?step=2')}>
-                Continue
-            </Button>
-        </div>
-    )
-}
-
-function StepTwo(): JSX.Element {
     const { personalizationData, step } = useValues(personalizationLogic)
     const { appendPersonalizationData } = useActions(personalizationLogic)
     const { reportPersonalizationSkipped, reportPersonalization } = useActions(eventUsageLogic)
 
-    const handleOptionChanged = (attr: 'role' | 'team_size', value: string | null): void => {
+    const handleOptionChanged = (attr: 'role' | 'products' | 'technical', value: string | string[] | null): void => {
         appendPersonalizationData({ [attr]: value })
     }
 
     const handleContinue = (): void => {
         reportPersonalization(personalizationData, step, answeredQuestionCount === TOTAL_QUESTION_COUNT)
         // TODO: Update organization record
-        // :TODO: Is there a way to force default insights graph this without hard reload?
+        // TODO: Is there a way to force default insights graph this without hard reload?
```

Comment:
Not a comment: :D Personal preference is to always use colon for these sort of comments.